bridges/GoogleScholarBridge.php

```diff
@@ -1,20 +1,102 @@
 <?php
 
-class GoogleScholarBridge extends BridgeAbstract
+class GoogleScholarV2Bridge extends BridgeAbstract
```

Comment:
Do not rename the existing bridge, otherwise existing subscriptions to it will break

Reply:
Another unintended bug, was rushing the changes. 👍